pkg/report/writer.go

```diff
@@ -116,8 +117,10 @@ func (tw TableWriter) write(result Result) {
 	}
 
 	var results []string
-	for _, severity := range dbTypes.SeverityNames {
-		r := fmt.Sprintf("%s: %d", severity, severityCount[severity])
+
+	for _, severity := range tw.Severities {
```

Comment:
```
./trivy image --severity MEDIUM,MEDIUM alpine:3.10.2                                                                              [~/src/github.com/aquasecurity/trivy]
2020-07-20T09:17:02.180+0300    INFO    Detecting Alpine vulnerabilities...

alpine:3.10.2 (alpine 3.10.2)
=============================
Total: 4 (MEDIUM: 4, MEDIUM: 4)
```

Could you unique `tw.Severities`? I recommend defining a new function like `Unique` or `UniqueStrings` in utils.
https://github.com/aquasecurity/trivy/blob/master/pkg/utils/utils.go

```suggestion
	for _, severity := range utils.Unique(tw.Severities) {
```

Reply:
Oke will do

Reply:
```
./trivy image --severity MEDIUM,MEDIUM,MEDIUM,HIGH,HIGH alpine:3.10.2
2020-07-20T19:57:37.220+0700    INFO    Detecting Alpine vulnerabilities...

alpine:3.10.2 (alpine 3.10.2)
=============================
Total: 4 (MEDIUM: 4, HIGH: 0)
```